torch/lib/THD/master_worker/worker/dispatch/Tensor.cpp

```diff
@@ -89,8 +89,8 @@ static void tensorNewWithStorage(rpc::RPCMessage& raw_message) {
   THLongStorage *stride = unpackTHLongStorage(raw_message);
   finalize(raw_message);
 
-  at::IntList sz(size->data, size->size);
```

Comment:
I was thinking about the change to remove `THTensor` and replace with ATen that is coming down the pipe, and when that change happens we're going to obliterate all of the data members in the struct and so it's better to access these members using a function rather than a direct member access.

If you would like, I can add some C++ methods to the struct so that I can rewrite this as `size->sizes()` and then include the `hpp` header to get at them (the methods could then be rewritten to forward to ATen when we get there.)

Reply:
This is actually pretty annoying to do because you can't name a method the same as a member, so if an existing member has taken the function name you wanted you have to do a codemod to update all of them. Probably best to batch this up with the movement to ATen.